commands/ping.go

```diff
@@ -0,0 +1,43 @@
+package commands
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+)
+
+var (
+	// PingCommand is the "ping" CLI command
+	PingCommand = &cobra.Command{
+		Use:   "ping",
+		Short: "Check API connection",
+		Long:  `Tests the connection to the API`,
+		Run:   ping,
+	}
+)
+
+// ping checks the API connections
+func ping(cmd *cobra.Command, args []string) {
+	uri := "https://api.giantswarm.io/v1/ping"
+	start := time.Now()
+	resp, err := http.Get(uri)
+	if err != nil {
+		fmt.Println(color.RedString("API cannot be reached"))
+		fmt.Println(err.Error())
+		os.Exit(1)
+	} else {
+		defer resp.Body.Close()
```

Comment:
I'd elide this else, as you're already returning inside the `err != nil` block

so, like:
```
if err != nil {
   ...
   os.Exit(1)
}

defer resp.Body.Close()
...
```